### PR TITLE
refactor: Clean-up post segregation of classic and enhanced meshing w…

### PR DESCRIPTION
**Pull request by prmukherj**
Clean-up the Enhanced meshing workflow post segregation from the Classic one. The "_dynamic_interface" flag is no longer to be required.

The "time.sleep(1)" is no longer needed as well now.

### CODE REVIEW

File: src/ansys/fluent/core/workflow.py [hunk 1]
```diff
@@ -477,44 +468,43 @@ def delete(self) -> None:
...
-                        self.python_name(), None
-                    )
-                )
-                self._command_source._python_name_display_text_map.pop(
-                    self.python_name(), None
-                )
-                self._command_source._repeated_task_python_name_display_text_map.pop(
+        self._command_source._dynamic_python_names = True
+        if (
+            self.python_name()
```
Comment (mkundu1) [inline src/ansys/fluent/core/workflow.py:473]: Shall we capture python_name in a local variable and use it throughout this function?

File: src/ansys/fluent/core/workflow.py [hunk 2]
```diff
@@ -1242,12 +1181,10 @@ def add_child_and_update(self, state=None, defer_update=None):
...
             self._task.AddChildAndUpdate()
-        time.sleep(1)
-        self.tasks()[-1]._set_python_name(compound=True)
-        return self.last_child()
+        return self._last_child()
 
-    def last_child(self) -> BaseTask:
-        """Get the last child of this CompoundTask.
+    def _last_child(self) -> BaseTask:
+        """Get the last child of this CompoundTask and formulate their python name.
```
Comment (raph-luc) [inline src/ansys/fluent/core/workflow.py:1187]:
```suggestion
        """Get the last child of this CompoundTask and set their Python name.
```

File: src/ansys/fluent/core/workflow.py [hunk 3]
```diff
@@ -1256,8 +1193,23 @@ def last_child(self) -> BaseTask:
         """
         children = self.tasks()
         if children:
+            py_name = self._get_python_names_for_compound_child()
```
Comment (raph-luc) [inline src/ansys/fluent/core/workflow.py:1196]: Does `_get_python_names_for_compound_child` only work for the last child of a compound task?

**Review by raph-luc — APPROVED** [review]

File: src/ansys/fluent/core/workflow.py (hunk 3 above)
Comment (prmukherj) [reply src/ansys/fluent/core/workflow.py:1196]: @raph-luc, yes as this step always follows the insertion of a Compound child task.

File: src/ansys/fluent/core/workflow.py (hunk 1 above)
Comment (prmukherj) [reply src/ansys/fluent/core/workflow.py]: Done, Thank you.

**Pull request merged**